Compiler Type-Checking errors

Hi,

the compiler outputs some errors for this library, so it's not possible to use it when your CI/CD contains type checking.

```
> tsc --noEmit

node_modules/@digikare/nestjs-prom/dist/common/prom.utils.d.ts:10:7 - error TS2314: Generic type 'Metric' requires 1 type argument(s).

10 }) => client.Metric;
         ~~~~~~~~~~~~~

node_modules/@digikare/nestjs-prom/dist/common/prom.utils.d.ts:15:7 - error TS2314: Generic type 'Counter<T>' requires 1 type argument(s).

15 }) => client.Counter;
         ~~~~~~~~~~~~~~

node_modules/@digikare/nestjs-prom/dist/common/prom.utils.d.ts:20:7 - error TS2314: Generic type 'Gauge<T>' requires 1 type argument(s).

20 }) => client.Gauge;
         ~~~~~~~~~~~~

node_modules/@digikare/nestjs-prom/dist/common/prom.utils.d.ts:25:7 - error TS2314: Generic type 'Histogram<T>' requires 1 type argument(s).

25 }) => client.Histogram;
         ~~~~~~~~~~~~~~~~

node_modules/@digikare/nestjs-prom/dist/common/prom.utils.d.ts:30:7 - error TS2314: Generic type 'Summary<T>' requires 1 type argument(s).

30 }) => client.Summary;
         ~~~~~~~~~~~~~~

node_modules/@digikare/nestjs-prom/dist/interfaces/index.d.ts:4:44 - error TS2314: Generic type 'Counter<T>' requires 1 type argument(s).

4 export declare class CounterMetric extends PromClient.Counter {
                                             ~~~~~~~~~~~~~~~~~~

node_modules/@digikare/nestjs-prom/dist/interfaces/index.d.ts:6:42 - error TS2314: Generic type 'Gauge<T>' requires 1 type argument(s).

6 export declare class GaugeMetric extends PromClient.Gauge {
                                           ~~~~~~~~~~~~~~~~

node_modules/@digikare/nestjs-prom/dist/interfaces/index.d.ts:8:46 - error TS2314: Generic type 'Histogram<T>' requires 1 type argument(s).

8 export declare class HistogramMetric extends PromClient.Histogram {
                                               ~~~~~~~~~~~~~~~~~~~~

node_modules/@digikare/nestjs-prom/dist/interfaces/index.d.ts:10:44 - error TS2314: Generic type 'Summary<T>' requires 1 type argument(s).

10 export declare class SummaryMetric extends PromClient.Summary {
                                              ~~~~~~~~~~~~~~~~~~

node_modules/@digikare/nestjs-prom/dist/interfaces/metric.type.d.ts:14:20 - error TS2314: Generic type 'CounterConfiguration<T>' requires 1 type argument(s).

14     configuration: PromClient.CounterConfiguration;
                      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

node_modules/@digikare/nestjs-prom/dist/interfaces/metric.type.d.ts:18:20 - error TS2314: Generic type 'GaugeConfiguration<T>' requires 1 type argument(s).

18     configuration: PromClient.GaugeConfiguration;
                      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

node_modules/@digikare/nestjs-prom/dist/interfaces/metric.type.d.ts:22:20 - error TS2314: Generic type 'HistogramConfiguration<T>' requires 1 type argument(s).

22     configuration: PromClient.HistogramConfiguration;
                      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

node_modules/@digikare/nestjs-prom/dist/interfaces/metric.type.d.ts:26:20 - error TS2314: Generic type 'SummaryConfiguration<T>' requires 1 type argument(s).

26     configuration: PromClient.SummaryConfiguration;
                      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

node_modules/@digikare/nestjs-prom/dist/prom.module.d.ts:7:38 - error TS2314: Generic type 'CounterConfiguration<T>' requires 1 type argument(s).

7     static forCounter(configuration: client.CounterConfiguration): DynamicModule;
                                       ~~~~~~~~~~~~~~~~~~~~~~~~~~~

node_modules/@digikare/nestjs-prom/dist/prom.module.d.ts:8:36 - error TS2314: Generic type 'GaugeConfiguration<T>' requires 1 type argument(s).

8     static forGauge(configuration: client.GaugeConfiguration): DynamicModule;
                                     ~~~~~~~~~~~~~~~~~~~~~~~~~

node_modules/@digikare/nestjs-prom/dist/prom.module.d.ts:9:40 - error TS2314: Generic type 'HistogramConfiguration<T>' requires 1 type argument(s).

9     static forHistogram(configuration: client.HistogramConfiguration): DynamicModule;
                                         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

node_modules/@digikare/nestjs-prom/dist/prom.module.d.ts:10:38 - error TS2314: Generic type 'SummaryConfiguration<T>' requires 1 type argument(s).

10     static forSummary(configuration: client.SummaryConfiguration): DynamicModule;
                                        ~~~~~~~~~~~~~~~~~~~~~~~~~~~


Found 17 errors.
```

--- a/package.json
+++ b/package.json
@@ -19,7 +19,6 @@
     "@nestjs/common": "^6.5.3",
     "@nestjs/core": "^6.5.3",
     "@types/node": "^10.14.16",
-    "prom-client": "^12.0.0",
     "reflect-metadata": "^0.1.13",
     "rxjs": "^6.5.2",
     "ts-node": "^8.3.0",
@@ -31,7 +30,9 @@
     "reflect-metadata": "^0.1.13",
     "rxjs": "^6.5.2"
   },
-  "dependencies": {},
+  "dependencies": {
+    "prom-client": "^12.0.0"
+  },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/digikare/nestjs-prom.git"

